spec/compute/models/server_spec.rb

```diff
@@ -0,0 +1,196 @@
+require "spec_helper"
+require "fog/compute/models/server"
+
+describe Fog::Compute::Server do
+  before do
+    @server = Fog::Compute::Server.new
+  end
+
+  describe "#sshable?" do
+    before do
+      # I'm not sure why #sshable? depends on a method that's not defined except in implementing classes
```

Comment:
How about a method that raises `NotImplementedError`? That's what we usually do for abstract interfaces on my team. It basically maintains the same behavior as currently without hidden side effects.

OTOH, http://chrisstump.online/2016/03/23/stop-abusing-notimplementederror/